pkg/gce-pd-csi-driver/controller_test.go

```diff
@@ -346,14 +346,19 @@ func TestCreateVolumeArguments(t *testing.T) {
 			},
 		},
 		{
-			name: "fail with MULTI_NODE_MULTI_WRITER capability",
+			name: "success with block/MULTI_NODE_MULTI_WRITER capabilities",
```

Comment:
what about `fs/MULTI_NODE_MULTI_WRITER` should we fail?

Reply:
@nikhilkathare Any details here?

Reply:
@mattcary  : Hi matt, fs/MULTI_NODE_MULTI_WRITER has handled above as mount/MULTI_NODE_MULTI_WRITER capability.

Reply:
> @nikhilkathare Any details here?

Hi matt, fs/MULTI_NODE_MULTI_WRITER has handled above as mount/MULTI_NODE_MULTI_WRITER capability.